grafana-plugin/src/containers/AttachIncidentForm/AttachIncidentForm.tsx

```diff
@@ -4,13 +4,11 @@ import { SelectableValue } from '@grafana/data';
 import { Button, Field, HorizontalGroup, Icon, Modal } from '@grafana/ui';
 import cn from 'classnames/bind';
 import { observer } from 'mobx-react';
-import moment from 'moment';
-import Emoji from 'react-emoji-render';
+import moment from 'moment-timezone';
```

Comment:
`eslint` was complaining about importing from `moment` because `@grafana/eslint-config` defines `moment` in the `no-restricted-imports` rule [here](https://github.com/grafana/eslint-config-grafana/blob/master/index.js#L42).

I searched around on Slack and found a PR from another team where they fixed this by importing from `moment-timezone` (plus we already had a bunch of files importing from there instead).